docs/developers/v2/fork-yearn-ui.md

```diff
@@ -0,0 +1,229 @@
+# Fork Yearn UI
+
+This is a step-by-step guide on how to fork and customize [Macarena Finance](https://github.com/yearn/macarena-finance) which is a yearn UI made to be forked. Deploying your own UI makes you eligible to receive partner [profit-sharing](https://docs.yearn.finance/partners/introduction#profit-share-model) fees.
+
+*Table of Conent*
+- [Install required software](#install-required-software)
+- [Clone Macarena Finance](#clone-macarena-finance)
+- [Change Vaults Filter](#change-vaults-filter)
+- [Change Logo](#change-logo)
+- [Change Colors](#change-colors)
+- [Change SEO settings](#change-seo-settings)
+- [Apply to receive partner fees](#apply-to-receive-partner-fees)
+
+## Install required software
+
+To follow all steps you need to install the following dependencies:
+
+1. **git**
+    - https://git-scm.com/downloads
+2. **node.js**
+    - https://nodejs.org/en/download/
+3. Open a terminal window and set up Yarn (used to install dependencies)
+    - open terminal **Windows**: Windows + R -> type `cmd` -> Enter
+    - open terminal **Mac**: CMD + Space -> type `Terminal` -> Enter
+    - set up yearn: `npm install --global yarn`
+
+## Clone Macarena Finance
+
+1. Fork Macarena Finance
+    - https://github.com/yearn/macarena-finance/fork
+2. Open terminal and clone your fork
+    - `git clone git@github.com:YOUR_GITHUB_NAME/macarena-finance.git`
+    - `cd macarena-finance` to enter project folder after cloning
+3. Install dependencies 
+    - `yarn install`
+4. Run developer environment
+    - `yarn dev`
+
+## Change Vaults Filter
+
+For this example we will make an UI focused on stablecoins, so we will add only vaults that deal with stablecoins and allow users to filter them by collateralization properties
+
+* Open `contexts/useYearn.tsx` and edit `endorsedVaults` to contain the vaults you want. In this example we will add only vaults that receive stablecoins.
```

Comment:
you already said this in the line above "In this example we will add only vaults that receive stablecoins."